### PR TITLE
enh(Resource-status): Improve details panel compactness

### PR DESCRIPTION
**Pull request by bdauria**
## Description

This greatly improves the compactness of the Resource details:

Capture d’écran 2021-03-18 à 11 19 36

## Type of change

- [x] Patch fixing an issue (non-breaking change)

## Target serie

- [x] 21.04.x (master)

### CODE REVIEW

File: www/front_src/src/Resources/Details/tabs/Details/Card.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+
+import { Paper, makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  container: {
```
Comment (Thebarda) [inline www/front_src/src/Resources/Details/tabs/Details/Card.tsx:6]: This class in not used here

File: www/front_src/src/Resources/Details/tabs/Details/Card.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    height: '100%',
+  },
+  content: {
+    padding: theme.spacing(1, 2, 2, 2),
+  },
+  title: {
```
Comment (Thebarda) [inline www/front_src/src/Resources/Details/tabs/Details/Card.tsx:12]: This class in not used here

File: www/front_src/src/Resources/Details/tabs/Details/Card.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    display: 'flex',
+    gridGap: theme.spacing(1),
+  },
+  active: {
```
Comment (Thebarda) [inline www/front_src/src/Resources/Details/tabs/Details/Card.tsx:16]: This class in not used here

File: www/front_src/src/Resources/Details/tabs/Details/DetailsCard/index.tsx [hunk 4]
```diff
@@ -1,27 +1,52 @@
...
+import IconCheck from '@material-ui/icons/Check';
+
+import { labelActive } from '../../../../translatedLabels';
+import Card from '../Card';
+
+const useStyles = makeStyles((theme) => ({
+  container: {
+    height: '100%',
+  },
+  content: {
```
Comment (Thebarda) [inline www/front_src/src/Resources/Details/tabs/Details/DetailsCard/index.tsx:15]: This one is not used in this component

Comment (bdauria) [reply www/front_src/src/Resources/Details/tabs/Details/DetailsCard/index.tsx]: We might want to check out something like: https://github.com/jens-ox/eslint-plugin-material-ui-unused-classes

File: www/front_src/src/Resources/Details/tabs/Details/DetailsCard/cards.tsx [hunk 5]
```diff
@@ -24,187 +24,113 @@ import {
...
       field: details.last_status_change,
-    }),
-    getCheckLines({ label: labelLastCheck, field: details.last_check }),
-    getCheckLines({ label: labelNextCheck, field: details.next_check }),
+      line: <DetailsLine line={toDateTime(details.last_status_change)} />,
+    },
+    {
+      title: labelLastCheck,
+      field: details.last_check,
+      active: details.active_checks,
```
Comment (kduret) [inline www/front_src/src/Resources/Details/tabs/Details/DetailsCard/cards.tsx:80]: not related to this PR, but this information is a bit wrong
it should be useful to get a tile which indicated if the service is actively and/or passively checks
But, we can't know if the last or next checks ar passive or active

**Review by kduret — DISMISSED** [review]

**Review by kduret — DISMISSED** [review]

**Review by jdelpierre — DISMISSED** [review]

File: www/front_src/src/Resources/Details/tabs/Details/Card.tsx [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+    padding: theme.spacing(1, 2, 2, 2),
+  },
+}));
+
+interface Props {
+  children?: React.ReactNode;
+  className?: string;
+}
+
+const Card = ({ children, className }: Props): JSX.Element => {
```
Comment (Thebarda) [inline www/front_src/src/Resources/Details/tabs/Details/Card.tsx:16]: A `Card` component already exist in Material UI https://material-ui.com/components/cards/#card. I think it is beter to use it rather create a new one that display the same layout

**Review by Thebarda — APPROVED** [review]

File: www/front_src/src/Resources/Details/tabs/Details/Card.tsx (hunk 6 above)
Comment (bdauria) [reply www/front_src/src/Resources/Details/tabs/Details/Card.tsx:16]: I know it does, I actually used it before this change, but it adds some extra spacing that I tried to reduce by coming up with our own one.

**Review by jdelpierre — APPROVED** [review]

**Pull request merged**